test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ModelExpressionTagHelper.cshtml

```diff
@@ -1,6 +1,6 @@
-@model DateTime
+@model DateTime?
```

Comment:
Hmmmm. That's a bug in our generated code if that's the case. It found a bugg!!!